package.json

```diff
@@ -36,7 +36,7 @@
         "@algolia/autocomplete-theme-classic": "1.12.1",
```

Comment:
It's a good occasion to upgrade Autocomplete too.
Actually I did it in https://github.com/algolia/algoliasearch-sfcc-b2c/pull/180, so if you merge `develop` it should be updated to the latest version.